src/Plugin/GraphQL/Schema/SdlSchemaPluginBase.php

```diff
@@ -41,6 +41,13 @@ abstract class SdlSchemaPluginBase extends PluginBase implements SchemaPluginInt
    */
   protected $inDevelopment;
 
+  /**
+   * The ID of the server using this plugin.
+   *
+   * @var string $serverId
```

Comment:
```suggestion
   * @var string
```

Will satisfy PHPCS (and should also satisfy PHPStan as can be seen on line 40 of this file). Class member variable declarations don't need a name specified in the docblock because it's unambigious (the comment is already linked to the variable declaration which includes the name).